static_src/test/unit/stores/app_store.spec.js

```diff
@@ -262,12 +262,19 @@ describe('AppStore', function() {
   });
 
   describe('on app restarted', function() {
-    it('should emit a change', function() {
-      const spy = sandbox.spy(AppStore, 'emitChange');
+    it('should poll until running instances is greater then 0', function() {
```

Comment:
Should this fire an action on resolve? Maybe the app received action shouldn't emit a change?